forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabMergeRequest.java

```diff
@@ -530,7 +530,7 @@ private String metadataFor(Check check) {
     }
 
     private String linkToDiff(String path, Hash hash, int line) {
-        return "[" + path + " line " + line + "](" + URIBuilder.base(repository.url())
+        return "[" + path + " line " + line + "](" + URIBuilder.base(repository.authenticatedUrl())
```

Comment:
This looks like another case of `.url()`. The authentication is even removed below. This string is built to be presented as text somewhere.